lib/stat_tracker.rb

```diff
@@ -197,6 +193,13 @@ def fewest_tackles(season)
   end
 
   ### HELPER METHODS ###
+  def percentage(stats)
+    (stats / number_of_games).round(2)
+  end
+  def number_of_games
```

Comment:
I need space...

Reply:
Just to clarify what Myles is referencing, I think he'se looking for a space at 198.5